### PR TITLE
Issue #17 POC demonstration plan

### PR DESCRIPTION
**Pull request by stoiyeet**
﻿## GitHub Issue

Issue #17

## Description

Added Problem statement to DevelopmentPlan tex and pdf

## How I tested

Rendered PDF locally to ensure correct formatting

### CODE REVIEW

File: docs/DevelopmentPlan/DevelopmentPlan.tex [hunk 1]
```diff
@@ -182,9 +182,7 @@ \section{Project Decomposition and Scheduling}
...
 
-What is the main risk, or risks, for the success of your project?  What will you
-demonstrate during your proof of concept demonstration to convince yourself that
-you will be able to overcome this risk?
+The proof of concept will be a demonstration of the team’s ability to develop the most crucial aspect of the final product, which is to take in a single note, and output the note and note length. If successfully performed it will clearly show that a viable product can be produced given enough time. The main risk associated with the success of the project is potential nuance introduced by human performance on instruments. Determining the time signature and Differentiating between notes n# and (n+1)♭  could potentially difficult tasks. It may become necessary to test and benchmark accuracy during audio edge cases. The Proof of concept will help dispel concerns by allowing the team to show the accuracy in audio recording of which the product will be capable. Given the ability to accurately take in music data, the completion of the product will be reduced to performance optimization and accuracy, construction of a user interface, and development of more advanced features.
```
Comment (emilyperica) [inline docs/DevelopmentPlan/DevelopmentPlan.tex:185]: 1. Standardize capitalizations (i.e. 'Proof of concept', 'signature and Differentiating')
2. 'Output the note and ...' -> 'Output the note _pitch_ and ...'
3. I know what you mean with n# and (n+1)b but it took me a second to understand, maybe add an example for clarity
4. 'could potentially difficult' -> 'could _be_ potentially difficult'; you also have two spaces after the flat 
5. Since we almost 100% will be testing accuracy of note detection you can change that line from 'we may' to 'we will'
6. We aren't developing a product for audio recording, change that to signal processing maybe
7. 'Take in music data' -> 'process audio'
8. Could you clarify the last sentence? Not entirely sure what it is saying

Just for overall readability of the .tex, could you also add a few line breaks here?

**Review by emilyperica — CHANGES_REQUESTED** [review]

**Review by emilyperica — APPROVED** [review]
LGTM
